### ISSUE TITLE
cmdline module fails to parse last argument (sometimes)

### ISSUE THREAD

**Issue by jlowellwofford** [kraken-hpc/uinit#4]
With cmdline: 

```console
BOOT_IMAGE=vmlinuz console=tty1 root=/dev/ram0 initrd=initramfs.cpio.gz kraken.ip=10.11.12.11 kraken.net=24 kraken.id=123e4567-e89d-12d3-a456-425455440096 kraken.parent=10.11.12.1
```

`parent` does not get set.

Module outputs the following:

```console
2021/01/05 11:58:18 uinit: cmdline: setting: ip = 10.11.12.11
2021/01/05 11:58:18 uinit: cmdline: setting: net = 24
2021/01/05 11:58:18 uinit: cmdline: setting: id = 123e4567-e89d-12d3-a456-425455440096
```

If a space is added at the end of the cmdline then `parent` is interpreted.

**Comment by jlowellwofford** [comment]
Caused by `SplitCommandLine` not trimming whitespace (specifically, newline in this case).

**Comment by jlowellwofford** [comment]
This was resolved by a5a9b4f1edb7bf0796cb8ed10c92d5296d362705
